packages/components/src/Alert/docs/pending.tsx

```diff
@@ -0,0 +1,33 @@
+import { Alert, AlertTrigger, Button, Content, Heading } from "@hopper-ui/components";
+import { useState } from "react";
+
+export default function Example() {
+    const [isOpen, setIsOpen] = useState(false);
+
+    const onClose = () => {
+        setIsOpen(false);
+    };
+
+    const handlePrimaryButtonClick = async() => {
+        await new Promise(resolve => setTimeout(resolve, 3000));
+        setIsOpen(false);
```

Comment:
you might be able to remove the setIsOpen here, since onClose will be called after once we switch the order in the function chain (see next comment)